.travis.yml

```diff
@@ -22,8 +22,8 @@ stages:
   - name: validate
   - name: deploy-develop
     if: type = push AND branch = main
-#  - name: deploy-prod
-#    if: type = push AND branch = main
+  - name: deploy-prod
+    if: type = push AND branch = main
```

Comment:
We don't have a main branch, the agora repo has develop, staging, and prod branches that correspond to the respective environments

Reply:
@JessterB you are thinking about `agora-infra` repo.  take note that this repo is a different repo, `agor2-infra`

Reply:
You are right, apologies!